zha/application/platforms/number/__init__.py

```diff
@@ -141,14 +141,13 @@ def native_step(self) -> float | None:
         """Return the value step."""
         return self._analog_output_cluster_handler.resolution
 
-    @functools.cached_property
+    @property
```

Comment:
To confirm, you changed this intentionally because the ZCL `description` attribute could theoretically change?

Reply:
Our use of `property` vs `cached_property` is a little inconsistent right now so I think either one works. Unit tests aren't affected so I'll revert the change.